### PR TITLE
better integration with PR API 

### LINKED ISSUE #5: Leave Off Unneeded Information
This is a recommendation from the Security and Privacy Checklist review. See https://docs.google.com/document/d/1w7ginyzNg-xZUmITK4vzcGUKB4gbMOAvlkWWaRtX14k/edit?usp=sharing for additional detail

The Basic Card Payment specification provides additional information that forms a unique correlator. Unlike unique, cross-origin information provided by the Payment Request API, the cardNumber field provided by the Basic Card API is (necessarily) required rather than optional. The current document appears to assume that any Basic Card Payment app will request all possible fields; however, there is [a PR filed for leaving off unneeded information](https://github.com/w3c/webpayments-methods-card/pull/4). For the sake of privacy, we recommend this PR be accepted.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Complement of https://github.com/w3c/payment-request/pull/700
redact phone number (closes #5)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/pull/52.html" title="Last updated on May 4, 2018, 1:06 AM GMT (e507b37)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/52/f3dc3d0...e507b37.html" title="Last updated on May 4, 2018, 1:06 AM GMT (e507b37)">Diff</a>

### CODE REVIEW

**Review by rsolomakhin — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -222,78 +223,174 @@ <h2>
...
+          item in <var>request</var>["supportedTypes"] to <var>types</var>.
+          </li>
+          <li>If <var>request</var>["supportedNetworks"] is present, append
+          each item in <var>request</var>["supportedNetworks"] to
+          <var>networks</var>.
+          </li>
+          <li>For each <var>card</var> in <var>cards</var>:
+            <ol>
+              <li>If <a>instrument is supported</a> <var>card</var>,
+              <var>networks</var>, <var>types</var> returns true, return true.
```
Comment (ianbjacobs) [inline index.html:259]: Hi @marcoscaceres, line 258 looks to be missing some punctuation (I think).

**Comment by ianbjacobs** [comment]
@marcoscaceres, sorry to have missed your review request. I read tonight and on first pass it looks good.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:259]: thanks! Will fix.

Comment (marcoscaceres) [reply index.html:259]: https://github.com/w3c/payment-method-basic-card/pull/54

**Pull request merged**